cppcache/src/PdxType.hpp

```diff
@@ -202,11 +202,38 @@ class PdxType : public internal::DataSerializableInternal,
 
   bool Equals(std::shared_ptr<PdxType> otherObj);
 
-  // This is for PdxType as key in std map.
-  bool operator<(const PdxType& other) const;
+  bool operator==(const PdxType& other) const;
+
+  size_t hashcode() const;
```

Comment:
Now that `std::hash` has the implementation is it necessary to have this on the class declaration? I don't see it in defined in the cpp file anymore.

Reply:
Good catch, im removing it